include/fizzy/fizzy.h

```diff
@@ -268,6 +268,15 @@ size_t fizzy_get_instance_memory_size(FizzyInstance* instance);
 bool fizzy_find_exported_table(
     FizzyInstance* instance, const char* name, FizzyExternalTable* out_table);
 
+/// Find exported memory by name.
+///
+/// @param  instance        Pointer to instance.
+/// @param  name            The table name. NULL-terminated string. Cannot be NULL.
+/// @param  out_memory      Pointer to output struct to store found memory. Cannot be NULL.
+/// @returns                true if memory was found, false otherwise.
+
```

Comment:
Should leave a comment this support a singular exported memory as per wasm 1.0.

Reply:
Well this function would support several memories just fine, because it finds them by name.